recipe/meta.yaml

```diff
@@ -14,22 +14,20 @@ build:
     - flake8heavened = flakeheaven:flake8_entrypoint
     - flakeheaven = flakeheaven:entrypoint
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
     - poetry
-    - python >=3.7.0
+    - python >=3.8.0
   run:
     - colorama
     - entrypoints
     - flake8 >=4.0.1,<5.0.0
-    # Only required for [py<38] but always include so this can be noarch: python
-    - importlib-metadata >=1.0,<2.0
```

Comment:
Needed to remove this upper pin to resolve dependency conflicts with packages that require `importlib_metadata>3`, e.g. `dvc`:

```
Could not solve for environment specs
The following packages are incompatible
├─ dvc >3.22  is installable and it requires
│  └─ dvc-task >=0.3.0,<1 , which requires
│     └─ celery >=5.3.0,<6 , which requires
│        └─ importlib-metadata >=3.6  with the potential options
│           ├─ importlib-metadata [1.1.3|1.5.0|...|4.8.1] would require
│           │  └─ python >=3.6,<3.7.0a0 , which can be installed;
│           ├─ importlib-metadata [1.1.3|1.5.0|...|4.9.0] would require
│           │  └─ python >=3.7,<3.8.0a0 , which can be installed;
│           ├─ importlib-metadata [1.1.3|1.5.0|...|4.9.0] would require
│           │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│           ├─ importlib-metadata [3.10.0|3.10.1|...|4.9.0] would require
│           │  └─ python >=3.9,<3.10.0a0 , which can be installed;
│           ├─ importlib-metadata [4.10.0|4.10.1|...|4.9.0] would require
│           │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│           └─ importlib-metadata [4.13.0|5.0.0|...|6.8.0], which can be installed;
└─ flakeheaven is installable and it requires
   └─ importlib-metadata >=1.0,<2.0  with the potential options
      ├─ importlib-metadata 1.7.0 conflicts with any installable versions previously reported;
      ├─ importlib-metadata [1.1.3|1.5.0] would require
      │  └─ python >=2.7,<2.8.0a0 , which can be installed;
      ├─ importlib-metadata [1.1.3|1.5.0|...|4.8.1], which can be installed (as previously explained);
      ├─ importlib-metadata [1.1.3|1.5.0|...|4.9.0], which can be installed (as previously explained);
      └─ importlib-metadata [1.1.3|1.5.0|...|4.9.0], which can be installed (as previously explained).
```